### DISCUSSION TITLE
Farming on one single node with plots from two different wallets?

### DISCUSSION THREAD

**Discussion by pjtham** [Chia-Network/chia-blockchain#6153] [Support]
Hi,

I opened 2 wallets, and two plots batches, with 2 different wallets signatures, on 2 machines.
One machine is dead, I want to "farm" its plots on the other one.

Can I actually farm all of the plots on one same machine with only one farmer (GUI), pointing rewards to a third (offline) wallet only from the GUI?

So far, I added the the new plots directory of the second wallet, and also added its 24 mnemonic into the GUI, but plots do not seem to appear in the farm overview, and the old wallet is not synced.

Thanks for your help.

**Comment by pjtham** [comment]
OK... My bad, restarting the full Chia GUI made the trick, everything works now perfectly fine. 
You can close this issue.

**Comment by BrandtH22** [comment]
@pjtham , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
